van-gogh-gallery/jonas/1. page/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
```

Comment:
Don't use spaces in folder or file names, there may be a lot of bugs with it and very hard to debug)

Reply:
FDdfgdfg